reference/tidy/setup.xml

```diff
@@ -11,7 +11,10 @@
   <para>
    Aby używać rozszerzenia Tidy należy zainstalować bibliotekę
    libTidy, która jest dostępna na witrynie
-   <link xlink:href="&url.tidy;">&url.tidy;</link>.
+   <link xlink:href="&url.tidy;">tidy homepage</link>.
+   Od PHP 7.1.0, następca obsługujący HTML5 biblioteka
+   <link xlink:href="&url.tidy5;">libtidy5</link> może być użyta alternatywnie.
+   Od PHP 7.3.0, inną opcją jest użycie biblioteki <link xlink:href="&url.tidyp;">libtidyp</link>.
```

Comment:
Niestety wyszło trochę nie po polsku, ciężko tego uniknąć kiedy masowo patrzy się na angielski szyk zdań. Powinno być coś w rodzaju "Od PHP 7.1.0 [brak przecinka] alternatywnie może być użyta biblioteka libtidy5, następca obsługujący HTML5".

Wtedy w polskiej wersji dwie linie będą zamienione miejscami, ale to bodaj jedyna opcja żeby całość brzmiała dobrze w naszym języku i osobiście nie widzę w tym problemu pod kątem późniejszego utrzymania tłumaczenia.